pkg/util/util.go

```diff
@@ -149,3 +149,16 @@ func isHostnameAllowed(hostname, allowedHost string) bool {
 
 	return false
 }
+
+// RemoveDuplicateStr removes duplicates from a slice of strings.
+func RemoveDuplicateStr(strSlice []string) []string {
+	allKeys := make(map[string]bool)
```

Comment:
We tend to use `map[string]struct{}` for sets, it's more efficient and idiomatic from what I've seen

Reply:
It won't work in this case as it will fail few lines below with:
```
'true' (type bool) cannot be represented by the type struct{}
```
I guess you were referring to `map[string]interface{}` maybe?